src/main/java/com/google/sps/data/Event.java

```diff
@@ -27,16 +27,18 @@
   private long id;
   // Represents the name of the Event.
   private String name;
-  // Holds ID (type long) of the Organization that owns the Event.
+  // Holds ID (type String) of the Organization that owns the Event.
   private long ownerOrgId;
+  // Holds name of the Organization that owns the Event. Is set in EventUpdater's setEventOwnerOrgName().
+  private String ownerOrgName;
   /* Holds ArrayList of Strings that represents an Organization's name. 
    * No need to store IDs of partnering organization, as partners will not have the privilege of editing Events.
    */
   private ArrayList<String> partnerNames;
   // Represents details of the Event. e.g. "In front of church ABC"
   private String details;
-  // Holds Date of Event and their corresponding time(s).
-  private HashMap<Date, EmbeddedEntity> dateAndHours;
+  // Holds Date of Event and their corresponding time(s). EmbeddedEntity holds a Date and an ArrayList of EmbeddedEntities that represents the time range(s) for that specific Date. This was chosen over a Map because Datastore does not support Maps.
```

Comment:
Is the purpose of this class to be convenient for:
    converting to gson?
    use in the servlets?
    modifying the datastore?

I would say that on class likely can't do all 3. If this object was for converting to gson, then the data and hours might actually be better used as a map. Let's call that out at the top.

Reply:
I believe this class is used for all three things listed. For the backend (use in servlet and modifying in datastore), it seems to make most sense to use ArrayList of embedded entities, especially because Datastore cannot store HashMaps. For the frontend, it does make more sense to use HashMaps, but we seem to not run into any problems when converting ArrayList of embedded entities using Gson. 

I'll add a comment!